.editorconfig

```diff
@@ -15,5 +15,8 @@ indent_size = 4
 [*.md]
 max_line_length = 80
 
+[*.svg]
+insert_final_newline = false
```

Comment:
not sure about this one, I'd prefer reverting it personnaly and adding the trailing LF

Reply:
I understand why, trailing spaces are added on the other file types and will prevent multiple line changes in git when lines are added. But for minified SVGs, things are a little different. Their content will always just be one line if they are minified. And even if they are not minified, markup will never get added to the end of the file because the file always ends with `</svg>` Additionally, trimming the last LF will make it possible to include the SVGs in twig js without trimming LF:
```html
<script>
  // Load icon in variable
  const symfonyIcon = "{{ include 'path/to/icon.svg' }}"

  // Append icon to body
  document.body.insertAdjacentHTML('beforeend', symfonyIcon)
</script>
```

Reply:
note that your example is a bad idea, as the SVG can totally contain a `"` which would break the JS...

Reply:
@stof, true, I've changed them to single quotes (also had single quotes for cases where I included them).

Reply:
the proper example would use escaping.

Reply:
I'll ditch the idea of removing the final newline, although it's a bit redundant, it might not be worth removing it since it introduces inconsistency over other file types. 

Thanks for the feedback on this guys!